.ci/generate_docs.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+BRANCH=$1
+FILES="docs/reference/ref.md docs/reference/repository.md docs/reference/system_schema.md docs/reference/woql.md"
+
+# Use the main branch of terminusdb if the branch is not a version or main itself
+if [[ $(echo "$BRANCH" | grep -vP "^main|^v") ]]; then
+   BRANCH="main"
+fi
+
+DOCS_DIR="$(pwd)"
+sudo apt install ronn
```

Comment:
Is this for https://github.com/terminusdb/terminusdb-docs/issues/17? Do you need to call `make docs` or something else?